src/client/components/heroImage/heroImage.css

```diff
@@ -0,0 +1,24 @@
+.hero-image {
+  height: 50%;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+.hero-text {
+  text-align: center;
+  position: absolute;
+  top: 40%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: white;
+}
+
+.title-text {
+  font-size: 50px;
+}
+
+#text {
```

Comment:
I couldn't find any element with id text in the component. So you can remove unused CSS  fx. text here

Reply:
Thanks for review comments.. I fixed it